react/features/recording/functions.js

```diff
@@ -65,7 +65,7 @@ export async function getRecordingLink(url: string, recordingSessionId: string,
     });
     const json = await res.json();
 
-    return res.ok ? json.url : Promise.reject(json);
+    return res.ok ? json : Promise.reject(json);
```

Comment:
nit: not related to this PR but we could do something like  : if (res.ok)  return json else throw json as we now mix async/await with promise approach.